fhir_2_ada-r4/zibs2020/payload/0.8.0-beta.1/all_zibs.xsl

```diff
@@ -26,27 +26,31 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
     <xsl:import href="contextContactEpisodeOfCare.xsl"/>
     <xsl:import href="ext-CopyIndicator.xsl"/>
     <xsl:import href="ext-MedicationAgreementPeriodOfUseCondition.xsl"/>
-    <xsl:import href="ext-TimeInterval-period.xsl"/>
     <xsl:import href="ext-TimeInterval-Duration.xsl"/>
+    <xsl:import href="ext-TimeInterval-period.xsl"/>
```

Comment:
See remarks in 0.0 version.

Reply:
Good question but out of scope for this ticket. Created new ticket: MP-1189